
Property changes on: .
___________________________________________________________________
Added: svn:ignore
   + _ReSharper.xVal
xVal.4.1.resharper.user
xVal.4.5.resharper.user
xVal.suo


Index: lib/aspnetmvc2/Microsoft.Web.Mvc.dll
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc2\Microsoft.Web.Mvc.dll
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc2/Microsoft.Web.Mvc.pdb
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc2\Microsoft.Web.Mvc.pdb
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc2/System.Web.Mvc.dll
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc2\System.Web.Mvc.dll
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc2/System.Web.Mvc.pdb
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc2\System.Web.Mvc.pdb
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc/Microsoft.Web.Mvc.dll
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc\Microsoft.Web.Mvc.dll
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc/Microsoft.Web.Mvc.pdb
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc\Microsoft.Web.Mvc.pdb
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc/System.Web.Abstractions.dll
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc\System.Web.Abstractions.dll
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc/System.Web.Mvc.dll
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc\System.Web.Mvc.dll
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc/System.Web.Mvc.pdb
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc\System.Web.Mvc.pdb
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: lib/aspnetmvc/System.Web.Routing.dll
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: lib\aspnetmvc\System.Web.Routing.dll
___________________________________________________________________
Added: svn:mime-type
   + application/octet-stream

Index: RulesProviders/CastleValidator/CastleValidator.csproj
===================================================================
--- RulesProviders/CastleValidator/CastleValidator.csproj	(revision 32458)
+++ RulesProviders/CastleValidator/CastleValidator.csproj	(working copy)
@@ -12,7 +12,7 @@
     <AssemblyName>xVal.RulesProviders.CastleValidator</AssemblyName>
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <SignAssembly>true</SignAssembly>
+    <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\xVal\xVal.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
Index: RulesProviders/NHibernateValidator/NHibernateValidator.csproj
===================================================================
--- RulesProviders/NHibernateValidator/NHibernateValidator.csproj	(revision 32458)
+++ RulesProviders/NHibernateValidator/NHibernateValidator.csproj	(working copy)
@@ -12,7 +12,7 @@
     <AssemblyName>xVal.RulesProviders.NHibernateValidator</AssemblyName>
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <SignAssembly>true</SignAssembly>
+    <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\xVal\xVal.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">

Property changes on: xVal
___________________________________________________________________
Added: svn:ignore
   + bin
obj



Property changes on: xVal.ClientSidePlugins
___________________________________________________________________
Added: svn:ignore
   + bin
obj
xVal.ClientSidePlugins.csproj.user


Index: xVal.ClientSidePlugins/Web.config
===================================================================
--- xVal.ClientSidePlugins/Web.config	(revision 32458)
+++ xVal.ClientSidePlugins/Web.config	(working copy)
@@ -38,7 +38,7 @@
 				<add assembly="System.Web.Extensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
 				<add assembly="System.Web.Abstractions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
 				<add assembly="System.Web.Routing, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
-				<add assembly="System.Web.Mvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
+				<add assembly="System.Web.Mvc, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null"/>
 				<add assembly="System.Data.DataSetExtensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089"/>
 				<add assembly="System.Xml.Linq, Version=3.5.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089"/>
 				<add assembly="System.Data.Linq, Version=3.5.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089"/>
@@ -103,7 +103,7 @@
 			<add verb="*" path="*.asmx" validate="false" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
 			<add verb="*" path="*_AppService.axd" validate="false" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
 			<add verb="GET,HEAD" path="ScriptResource.axd" type="System.Web.Handlers.ScriptResourceHandler, System.Web.Extensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" validate="false"/>
-			<add verb="*" path="*.mvc" validate="false" type="System.Web.Mvc.MvcHttpHandler, System.Web.Mvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
+			<add verb="*" path="*.mvc" validate="false" type="System.Web.Mvc.MvcHttpHandler, System.Web.Mvc, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null"/>
 		</httpHandlers>
 		<httpModules>
 			<add name="ScriptModule" type="System.Web.Handlers.ScriptModule, System.Web.Extensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
@@ -146,7 +146,7 @@
 			<add name="ScriptHandlerFactory" verb="*" path="*.asmx" preCondition="integratedMode" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
 			<add name="ScriptHandlerFactoryAppServices" verb="*" path="*_AppService.axd" preCondition="integratedMode" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
 			<add name="ScriptResource" preCondition="integratedMode" verb="GET,HEAD" path="ScriptResource.axd" type="System.Web.Handlers.ScriptResourceHandler, System.Web.Extensions, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
-			<add name="MvcHttpHandler" preCondition="integratedMode" verb="*" path="*.mvc" type="System.Web.Mvc.MvcHttpHandler, System.Web.Mvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
+			<add name="MvcHttpHandler" preCondition="integratedMode" verb="*" path="*.mvc" type="System.Web.Mvc.MvcHttpHandler, System.Web.Mvc, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null"/>
 			<add name="UrlRoutingHandler" preCondition="integratedMode" verb="*" path="UrlRouting.axd" type="System.Web.HttpForbiddenHandler, System.Web, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"/>
 		</handlers>
 	</system.webServer>
Index: xVal.ClientSidePlugins/xVal.ClientSidePlugins.csproj
===================================================================
--- xVal.ClientSidePlugins/xVal.ClientSidePlugins.csproj	(revision 32458)
+++ xVal.ClientSidePlugins/xVal.ClientSidePlugins.csproj	(working copy)
@@ -47,9 +47,9 @@
       <HintPath>..\..\..\..\..\..\Program Files (x86)\Microsoft ASP.NET\ASP.NET MVC Beta\Assemblies\System.Web.Abstractions.dll</HintPath>
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
-    <Reference Include="System.Web.Mvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="System.Web.Mvc, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\..\..\..\Program Files (x86)\Microsoft ASP.NET\ASP.NET MVC Beta\Assemblies\System.Web.Mvc.dll</HintPath>
+      <HintPath>..\lib\aspnetmvc2\System.Web.Mvc.dll</HintPath>
     </Reference>
     <Reference Include="System.Web.Routing, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
@@ -120,7 +120,7 @@
         <WebProjectProperties>
           <UseIIS>False</UseIIS>
           <AutoAssignPort>False</AutoAssignPort>
-          <DevelopmentServerPort>49792</DevelopmentServerPort>
+          <DevelopmentServerPort>3000</DevelopmentServerPort>
           <DevelopmentServerVPath>/</DevelopmentServerVPath>
           <IISUrl>
           </IISUrl>

Property changes on: xVal.Tests
___________________________________________________________________
Added: svn:ignore
   + bin
obj


Index: xVal.Tests/xVal.Tests.csproj
===================================================================
--- xVal.Tests/xVal.Tests.csproj	(revision 32458)
+++ xVal.Tests/xVal.Tests.csproj	(working copy)
@@ -53,7 +53,10 @@
     <Reference Include="System.Web.Abstractions">
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
-    <Reference Include="System.Web.Mvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+    <Reference Include="System.Web.Mvc, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\lib\aspnetmvc2\System.Web.Mvc.dll</HintPath>
+    </Reference>
     <Reference Include="System.Web.Routing">
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
Index: xVal/xVal.csproj
===================================================================
--- xVal/xVal.csproj	(revision 32458)
+++ xVal/xVal.csproj	(working copy)
@@ -12,7 +12,7 @@
     <AssemblyName>xVal</AssemblyName>
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <SignAssembly>true</SignAssembly>
+    <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>xVal.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
@@ -43,6 +43,10 @@
     <Reference Include="System.Web.Extensions">
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
+    <Reference Include="System.Web.Mvc, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\lib\aspnetmvc2\System.Web.Mvc.dll</HintPath>
+    </Reference>
     <Reference Include="System.Xml.Linq">
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
@@ -51,7 +55,6 @@
     </Reference>
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
-    <Reference Include="System.Web.Mvc" />
     <Reference Include="System.Web.Routing" />
     <Reference Include="System.Web.Abstractions" />
   </ItemGroup>
